src/explorer.ts

```diff
@@ -240,6 +246,9 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
             this.kind === kuberesources.allKinds.secret ||
             this.kind === kuberesources.allKinds.configMap) {
             treeItem.contextValue = `vsKubernetes.resource.${this.kind.abbreviation}`;
+            if (this.kind === kuberesources.allKinds.pod && this.metadata.status !== null) {
```

Comment:
For extra safety, consider `if (this.kind === … && this.metadata && this.metadata.status)`

Reply:
refactored the code to reflect this change.